Lib/idlelib/replace.py

```diff
@@ -44,37 +71,54 @@ def open(self, text):
         first = first or text.index("insert")
         last = last or first
         self.show_hit(first, last)
-        self.ok = 1
+        self.ok = True
 
     def create_entries(self):
-        """Create label and text entry widgets"""
+        "Create base and additional label and text entry widgets."
         SearchDialogBase.create_entries(self)
         self.replent = self.make_entry("Replace with:", self.replvar)[0]
 
     def create_command_buttons(self):
+        """Create base and additional command buttons.
+
+        The additional buttons are for Find, Replace,
+        Replace+Find, and Replace All.
+        """
         SearchDialogBase.create_command_buttons(self)
         self.make_button("Find", self.find_it)
         self.make_button("Replace", self.replace_it)
-        self.make_button("Replace+Find", self.default_command, 1)
+        self.make_button("Replace+Find", self.default_command, isdef=True)
         self.make_button("Replace All", self.replace_all)
 
     def find_it(self, event=None):
-        self.do_find(0)
+        """Handle the Find button.
+
+        In the Replace dialog, repeatedly using Find does not
+        go to the next occurrence of the pattern.
```

Comment:
It does for me.  If it does not on linux, it would be a bug to be fixed.

Reply:
This works for me on Windows and Linux now.  I'm not sure why I thought it didn't.  If I change the call to `self.do_find(True)`, then it won't go to the next occurrence, so maybe I had the flag set wrong when I was testing it.  Sorry about the false alarm.